libmscore/drumset.h

```diff
@@ -14,13 +14,26 @@
 #define __DRUMSET_H__
 
 #include "mscore.h"
+#include "tremolo.h"
 #include "note.h"
+#include "articulation.h"
 #include "sym.h"
 
 namespace Ms {
 
 class Xml;
 
+struct DrumInstrumentVariant {
+      int pitch;
+      QString articulationName;
+      TremoloType tremolo;
+      DrumInstrumentVariant() {
```

Comment:
It is better to use the following syntax to avoid assigning values twice:
```
DrumInstrumentVariant()
      : pitch(INVALID_PITCH),
      tremolo(TremoloType::INVALID_TREMOLO) {
```
QString is initialized with empty value in default constructor, so no need to assign empty string.

Reply:
Ok thanks !